test/services/user-group/index.test.js

```diff
@@ -84,6 +84,19 @@ describe('classes service', () => {
 			expect(data[0]._id.toString()).to.equal(classes[0]._id.toString());
 		});
 
+
+		it('CREATE patches successor ID in predecessor class', async () => {
+			const orgClass = await testObjects.createTestClass({ name: 'sonnenklasse 1' });
+			const successorClass = await testObjects.createTestClass({
```

Comment:
I would argue against using the test helpers here, because they are volatile (most of them use the models directly, for instance). Instead, use the method you want to test; `app.service('classes').create` in this case.

Reply:
updated it